models/__init__.py

```
@@ -0,0 +1,37 @@
+    "parse_id",
+    "CORE_THESIS",
+    "BackboneItem",
+    "Cluster",
+    "DomainMechanic",
+    "Framework",
+    "LearningPrereq",
+    "Metadata",
+    "ProvisionalMap",
+    "Relationships",
+    "Subnode",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)